engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/io/PrintlnNode.java

```diff
@@ -55,11 +55,10 @@ Object doPrint(
       Object message,
       @CachedLibrary(limit = "10") InteropLibrary strings,
       @Cached("buildSymbol()") UnresolvedSymbol symbol,
-      @Cached("buildInvokeCallableNode()") InvokeCallableNode invokeCallableNode,
-      @Cached ExpectStringNode expectStringNode) {
+      @Cached("buildInvokeCallableNode()") InvokeCallableNode invokeCallableNode) {
     var str = invokeCallableNode.execute(symbol, frame, state, new Object[] {message});
     EnsoContext ctx = EnsoContext.get(this);
-    print(ctx.getOut(), expectStringNode.execute(str));
+    print(ctx.getOut(), str);
```

Comment:
The `ExpectStringNode` was also dealing with stripping warnings from this value.

Now, if I understand it correctly, the warnings will be printed if present. Although tbh I did not test it yet - probably worth doing first.

Still, I know there were different opinions on this - are we ok with printing warnings, do we want to keep dropping them or do we want to print them to stderr?

cc: @jdunkerley @JaroslavTulach 

I like printing them like currently, we can always clear them if we do not want them in the output.

I'm not a fan of the `stderr` idea, because it may be unexpected (we should have a separate method to print to stderr) and it may be hard to keep track which warnings were associated with which value if many values are printed.

Maybe the best solution will be to duplicate `println` into a `IO.debug_print` node?

Then we could have `println` keep the existing behaviour, always expecting text returned from `to_text` and failing with type error if that invariant is broken. And `debug_print` would be more lenient about types and also would print warnings?

Reply:
Based on Rust print vs. debug print:

> Maybe the best solution will be to duplicate println into a IO.debug_print node?

```haskell
IO.println warnings=True "Hi with warnings"
```

Reply:
I think having the control to print warnings would be good - my feeling is we should print the value without warnings by default ad then have the option to show warnings.

This would be consistent with our visualizations in the IDE as well.

Reply:
Ok, that sounds good. I guess we need to add an `IO.println_builtin` to be able to add a default argument (since a builtin cannot directly have default arguments, right? cc: @JaroslavTulach or was this limitation lifted?) and I will make `warnings=False` by default.

I'll update this PR once I have some more free time, probably after I finish the full join PR.

Reply:
> we need to add an `IO.println_builtin` to be able to add a default argument (since a builtin cannot directly have default arguments, right? cc: @JaroslavTulach or was this limitation lifted?) 

Yes, separate builtin method is needed. The limitation is still there:
- #5312